go/internal/feast/server/http_server.go

```diff
@@ -0,0 +1,248 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/feast-dev/feast/go/internal/feast"
+	"github.com/feast-dev/feast/go/internal/feast/model"
+	"github.com/feast-dev/feast/go/internal/feast/server/logging"
+	prototypes "github.com/feast-dev/feast/go/protos/feast/types"
+	"net/http"
+)
+
+type httpServer struct {
+	fs             *feast.FeatureStore
+	loggingService *logging.LoggingService
+	server         *http.Server
+}
+
+// Some Feast types aren't supported during JSON conversion
+type repeatedValue struct {
+	stringVal     []string
+	int64Val      []int64
+	doubleVal     []float64
+	boolVal       []bool
+	stringListVal [][]string
+	int64ListVal  [][]int64
+	doubleListVal [][]float64
+	boolListVal   [][]bool
+}
+
+func (u *repeatedValue) UnmarshalJSON(data []byte) error {
```

Comment:
Would be great to have some unit tests on this function

Reply:
Added